regress/core/in_marc21_expected

```diff
@@ -0,0 +1,63 @@
+ERROR:  invalid MARC21/XML document.
+ERROR:  invalid MARC21/XML document. Root element <record> expected but <collection> found.
+ERROR:  invalid MARC21/XML document. Root element <record> expected but <collection> found.
+ERROR:  invalid MARC21/XML document.
+ERROR:  invalid MARC21/XML document.
+ERROR:  parse error - invalid literal at 034$d: "N42"
+ERROR:  parse error - invalid literal at 034$e: "N42"
+ERROR:  parse error - invalid literal at 034$f: "N42"
+ERROR:  parse error - invalid literal at 034$g: "N42"
+ERROR:  parse error - the Coded Cartographic Mathematical Data (datafield:034) in the given MARC21/XML is incomplete. Coordinates for subfields "$d","$e","$f" and "$g" are expected.
+ERROR:  parse error - the Coded Cartographic Mathematical Data (datafield:034) in the given MARC21/XML is incomplete. Coordinates for subfields "$d","$e","$f" and "$g" are expected.
+ERROR:  parse error - the Coded Cartographic Mathematical Data (datafield:034) in the given MARC21/XML is incomplete. Coordinates for subfields "$d","$e","$f" and "$g" are expected.
+ERROR:  parse error - the Coded Cartographic Mathematical Data (datafield:034) in the given MARC21/XML is incomplete. Coordinates for subfields "$d","$e","$f" and "$g" are expected.
+ERROR:  parse error - invalid literal at 034$d: "x1204600"
+ERROR:  parse error - invalid literal at 034$e: " 1202600"
+ERROR:  parse error - invalid literal at 034$f: "f0354700"
+ERROR:  parse error - invalid literal at 034$g: ".0354700"
+ERROR:  parse error - invalid literal at 034$e: "-W1202600"
+ERROR:  parse error - invalid literal at 034$f: "N+0354700"
+ERROR:  parse error - invalid literal at 034$g: "++0354700"
+ERROR:  parse error - invalid literal at 034$e: "--1202600"
+ERROR:  parse error - invalid literal at 034$d: "W150 4600"
+ERROR:  parse error - invalid literal at 034$f: "N035+4700"
+ERROR:  parse error - invalid literal at 034$d: "W150-4600"
+ERROR:  parse error - invalid literal at 034$g: "N048/4700"
+ERROR:  parse error - invalid literal at 034$g: "N048*4700"
+ERROR:  parse error - invalid literal at 034$d: "W120.46.89"
+ERROR:  parse error - invalid literal at 034$e: "W120.46,89"
+ERROR:  parse error - invalid literal at 034$f: "W120,46,89"
+ERROR:  parse error - invalid literal at 034$g: "W12,046.89"
+ERROR:  parse error - invalid literal at 034$d: ""
+ERROR:  parse error - invalid literal at 034$e: ""
+ERROR:  parse error - invalid literal at 034$f: ""
+ERROR:  parse error - invalid literal at 034$g: ""
+ERROR:  invalid MARC21/XML document.
+ERROR:  invalid MARC21/XML document.
+polygon_1|POLYGON((45.78333333333333 35.78333333333333,35.78333333333333 35.78333333333333,35.78333333333333 52.78333333333333,45.78333333333333 52.78333333333333,45.78333333333333 35.78333333333333))
```

Comment:
When generating WKT to compare in tests, wrapping in a roundoff like ST_AsTest(geom, 5) will make the test more stable, since presumably you don't really care about the terminal 10th decimal